css/contact.css

```diff
@@ -135,6 +135,82 @@ form {
 .bottom {
   order: 4;
   position: relative;
-  top: 20vh;
-  left: 18vh;
+  top: 18vh;
+  left: 12vh;
+}
+
+@media only screen and (min-width: 768px) {
+  #contact {
+    position: relative;
+    bottom: 20vh;
+  }
+
+  #form-wrapper {
+    width: 80%;
+    margin: auto;
+    padding: 0;
+    position: relative;
+    bottom: 5vh;
+  }
+
+  #contact-wrapper {
+    background: url("/Images/desktop/contact\ form\ background\ shapes@2x.png") no-repeat center center;
+    background-color: #6070ff;
+    background-size: cover;
+    height: 120vh;
```

Comment:
- [ ] As requested by the previous reviewer kindly fix the contact section as while making the latest changes you introduced a bug by mistake :recycle: 

![Screenshot from 2023-03-04 15-54-15](https://user-images.githubusercontent.com/97834160/222906603-d84a8e3a-0d1c-4c78-8951-46db952063b3.png)

Reply:
Bug has been fixed